ui/StatusQ/src/StatusQ/Core/Theme/Theme.qml

```diff
@@ -1,10 +1,9 @@
 pragma Singleton
 
-import QtQuick 2.13
+import QtQuick 2.15
+import QtQuick.Controls.Universal 2.15
```

Comment:
Do we import universal style only to use enum with `Light`/`Dark`/`System`? I know it's old thing but maybe it's good moment to define own enum and clean it up. Or maybe better to create a separate task :)

Reply:
Yeah that's the builtin style that we use internally; so I'd keep it that way

Reply:
ahh, `app/res/qtquickcontrols2.conf`, ok then